diff --git a/node_modules/react-native-search-bar/android/build.gradle b/node_modules/react-native-search-bar/android/build.gradle
index dd30d7b..04e4c26 100644
--- a/node_modules/react-native-search-bar/android/build.gradle
+++ b/node_modules/react-native-search-bar/android/build.gradle
@@ -7,6 +7,10 @@ def safeExtGet(prop, fallback) {
 android {
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
+    defaultConfig {
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+    }
+
     compileOptions {
         sourceCompatibility = '1.8'
         targetCompatibility = '1.8'