diff --git a/node_modules/@types/detox/runners/jest/assignReporter.d.ts b/node_modules/@types/detox/runners/jest/assignReporter.d.ts
new file mode 100644
index 0000000..ed29dbd
--- /dev/null
+++ b/node_modules/@types/detox/runners/jest/assignReporter.d.ts
@@ -0,0 +1,12 @@
+interface JasmineAssignReporter {
+    // These are not publicly used, but are defined in order to overlap with the jasmine.CustomReporter interface (which is a weak interface)
+    suiteStarted: () => void;
+    suiteDone: () => void;
+    assignDone: () => void;
+    assignStarted: () => void;
+}
+
+declare const assignReporter: JasmineAssignReporter;
+
+export = assignReporter;
+
